This patch fixes gcc-4.1.2's mishandling of the texinfo version numbers so that
the gcc build detects newer versions of texinfo correctly.
--- configure	2006-11-21 17:48:36.000000000 +0000
+++ configure-new	2010-12-30 18:55:12.950488002 +0000
@@ -3549,7 +3549,7 @@
     # For an installed makeinfo, we require it to be from texinfo 4.2 or
     # higher, else we use the "missing" dummy.
     if ${MAKEINFO} --version \
-       | egrep 'texinfo[^0-9]*([1-3][0-9]|4\.[2-9]|[5-9])' >/dev/null 2>&1; then
+       | egrep 'texinfo[^0-9]*(4\.([4-9]|[1-9][0-9])|[5-9]|[1-9][0-9])' >/dev/null 2>&1; then
       :
     else
       MAKEINFO="$MISSING makeinfo"
